homeassistant/components/stream/core.py

```diff
@@ -19,12 +18,13 @@
 PROVIDERS = Registry()
 
 
-@attr.s
+@attr.s(slots=True)
 class Segment:
     """Represent a segment."""
 
     sequence: int = attr.ib()
-    segment: io.BytesIO = attr.ib()
+    init: bytes = attr.ib()
```

Comment:
Could add a comment here describing that init + moof_data is the video segment payload.